tests/migration_test.go

```diff
@@ -336,14 +336,14 @@ var _ = Describe("[Serial][rfe_id:393][crit:high][vendor:cnv-qe@redhat.com][leve
 
 			// verify it is numeric
 			_, err = strconv.Atoi(words[0])
-			Expect(err).ToNot(HaveOccurred(), "should have found pid for libvirtd that is numeric")
 
 			pid = words[0]
+			Expect(err).ToNot(HaveOccurred(), fmt.Sprintf(errorMassageFormat, stdout, stderr, errPsCmd, pid))
 			break
 
 		}
+		Expect(pid).ToNot(Equal(""), fmt.Sprintf(errorMassageFormat, stdout, stderr, errPsCmd, pid))
```

Comment:
nit: you can also write it this way: `Expect(pid).ToNot(BeEmpty(), ...)`, a bit more declarative :)

Reply:
done